### DISCUSSION TITLE
Data processing error

### DISCUSSION THREAD

**Discussion by arthurlalayan** [gjoseph92/stackstac#190] [Q&A]
I am using dask distributed and stackstac. During data processing I am getting the following error: 
![image](https://user-images.githubusercontent.com/43277398/201594889-88f9ac1b-a416-40ef-8bff-abf36a5e50a9.png)
Could anyone help me?

**Comment by gjoseph92** [comment]
@arthurlalayan please see https://distributed.dask.org/en/stable/killed.html for suggestions diagnosing these sorts of issues. This is not necessarily a stackstac bug, it could also just be your workers are running out of memory or many other issues. You'll need to look at worker logs at least to start diagnosing this.

Also, looks like you're using an older version of dask. You should consider upgrading to the latest version being released today https://github.com/dask/community/issues/287.
